README.md

```diff
@@ -1,2 +1,73 @@
-# CORE
 [![Build Status](https://travis-ci.org/mentorpaired/core.svg?branch=staging)](https://travis-ci.org/mentorpaired/core) [![Coverage Status](https://coveralls.io/repos/github/mentorpaired/core/badge.svg?branch=staging)](https://coveralls.io/github/mentorpaired/core?branch=staging)
+
+# CORE
+Core is the backend of MentorPaired, an app which connects software engineering mentors to mentees who seek to learn any programming language.
+
+### Tech
+Core is written in Python3 and Django 3.0.3.
+
+### Installation For Ubuntu 18.04 Users
+
+ Please install and set up the following packages first, upgrade if you find the package is already installed:
+* Python3. Run the 'python3 -V' command to see the version you have installed.
+* PostgreSQL (Ensure the server is running). Alternatively, if your project is a small one or you find you don't need PostgreSQL, Python includes a lightweight database called SQLite so you won't need to set up a database, however it cannot support a high level of concurrency.
+
+* It is advisable to install Django in a virtual environment and the library for doing this is called [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation). Install this library with:
+```
+sudo pip install virtualenvwrapper
```

Comment:
You don't need to add sudo. They can add sudo if they have permissions issue, not right away.